.travis.yml

```diff
@@ -21,6 +24,8 @@ jobs:
       install: *ccache-setup-steps
       script:
         - pyenv global 2.7.15
+        - pip install --upgrade pip
+        - pip install six
```

Comment:
I think you can remove this now you've included it right?

Reply:
Good catch.  Done.